package.json

```diff
@@ -19,6 +19,7 @@
     "test": "jest ."
   },
   "peerDependencies": {
+    "babel-eslint": "^10.0.0",
```

Comment:
We're using `babel` as the parser in the [base config](https://github.com/ebates-inc/eslint-config/blob/master/index.js#L12) so it should be a dependency.